src/pybind/mgr/dashboard/frontend/src/app/shared/services/task-message.service.ts

```diff
@@ -539,6 +543,10 @@ export class TaskMessageService {
     }'`;
   }
 
+  smb(metadata: any) {
```

Comment:
```suggestion
  smb(metadata: {cluster_id: string}) {
```
We are missing the types in entire file but we can start from now adding them.
Thanks for patience and gr8 work here @dnyanee1997 ❤️

Reply:
@afreen23 , updated the PR.
Thanks for the approval!